### ISSUE TITLE
Family emojis not supported by Emoji Tools?

### ISSUE THREAD

**Issue by MrYawnie** [MitchTalmadge/Emoji-Tools#26]
Hello again Mitch,

I tried your updated Emoji Tools version for the first time, and tried to add family emojis to my font file automatically, rather than manually as I did before your update (remember?). It was quite tedious to manually add 100 unseen emojis to the font file. I understood v1.91 supports automatic adding of new emojis.

It however, did not add emojis with names such as "uni1f467_uni200d_uni1f467_uni200d_uni1f468_uni200d_uni1f468.png" which is man, man, girl, girl in one emoji image.

Could you look into the issue? When i generated the new font file, and then extracted it, the family emoji images simply weren't added at all.

**Comment by MrYawnie** [comment]
Edit: I saw the other posts that the latest version doesn't support adding ligature sets, so that explained my issue. Hopefully you'll find a solution soon :)

**Comment by ayrton2510** [comment]
Download latest Android N emoji font. It includes unicode 9.0.

**Comment by MrYawnie** [comment]
I did, but when i tried to change the emojis to iOS emojis, the packaging worked, but none of the emojis were visible. Just blank space. Adding new emojis didn't work for me either, the same thing happened. The emojis are in the font file and in the .ttx file, but they're not visible for some reason.

**Comment by ayrton2510** [comment]
Did you remember to edit the ttx file? If you have a different PNG size (160x160 for example), you need to edit the ttx.

**Comment by MrYawnie** [comment]
Yeah, forgot that. The issue is to start editing some 1600+ entries with the size :D Replace all won't work as there's 1800+ emojis in Android N emoji file. Which means more than 200 emojis wouldn't work after that.. Changing them back to original size in ttx file would also take a lot of time manually.
